internal/services/compute/virtual_machine.go

```diff
@@ -284,8 +301,14 @@ func flattenVirtualMachineOSDisk(ctx context.Context, disksClient *compute.Disks
 
 	diffDiskSettings := make([]interface{}, 0)
 	if input.DiffDiskSettings != nil {
+		placement := string(compute.DiffDiskPlacementCacheDisk)
+		if input.DiffDiskSettings.Placement != "" {
+			placement = string(input.DiffDiskSettings.Placement)
+		}
+
 		diffDiskSettings = append(diffDiskSettings, map[string]interface{}{
-			"option": string(input.DiffDiskSettings.Option),
+			"option":    string(input.DiffDiskSettings.Option),
+			"placement": placement,
```

Comment:
Can we set the placement as what it is returned?
```suggestion
			"placement": string(input.DiffDiskSettings.Placement),
```